Example/WaypointConfirmationViewController.swift

```diff
@@ -1,13 +0,0 @@
-import UIKit
-
-protocol WaypointConfirmationViewControllerDelegate: NSObjectProtocol {
```

Comment:
Not used anywhere at this point. We also have standalone example here in https://github.com/mapbox/mapbox-navigation-ios-examples/blob/main/Navigation-Examples/Examples/Waypoint-Arrival-Screen.swift.